ts.config.js

```diff
@@ -0,0 +1,16 @@
+{
+  "compilerOptions": {
+    "jsx": "react",
+    "module": "commonjs",
+    "target": "es6",
+    "allowJs": true,
+    "rootDir": "src",
+    "outDir": "dist",
+    "sourceMap": true,
+    "strict": false,
+    "esModuleInterop": true,
+    "moduleResolution": "node"
+  },
+  "include": ["src"],
+  "exclude": ["dist", "node_modules"]
```

Comment:
Exclude is only to exclude paths that are within the include paths. Unless `dist` and `node_modules` are under `src`, this exclude is not needed.